### PR TITLE
Add metering archive permissions to helm chart

### PR DESCRIPTION
**Pull request by smarlowucf**

### CODE REVIEW

File: data/overlayfiles/rancher-deployer/cnab/app/charts/csp-rancher-usage-operator/100.0.0+up0.1.0/templates/rbac.yaml [hunk 1]
```diff
@@ -86,6 +86,20 @@ rules:
   - {{ template "csp-usage-operator.cspBillingAdapterConfigMap"  }}
   verbs:
   - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Chart.Name }}-archive-reader-role
```
Comment (guangyee) [inline data/overlayfiles/rancher-deployer/cnab/app/charts/csp-rancher-usage-operator/100.0.0+up0.1.0/templates/rbac.yaml:93]: Do we really need two separate roles for this? Can't we just add the resource to the configmap-reader-role?

**Comment by guangyee** [comment]
Also, I don't see csp-rancher-usage-operator code accessing the archive. Where is it being used?

**Comment by smarlowucf** [comment]
> Also, I don't see csp-rancher-usage-operator code accessing the archive. Where is it being used?

Archive feature would be triggered on the first billing cycle.

**Comment by smarlowucf** [comment]
Combined the config maps

**Review by guangyee — APPROVED** [review]
lgtm

**Pull request merged**